src/toil/provisioners/abstractProvisioner.py

```diff
@@ -778,11 +779,19 @@ def addToilService(self, config: InstanceConfiguration, role: str, keyPath: str
             WantedBy=multi-user.target
             '''))
 
+    @staticmethod
+    def getArchitecture():
+        # https://stackoverflow.com/questions/45125516/possible-values-for-uname-m
+        if platform.machine() in ['arm', 'aarch64_be', 'aarch64', 'armv8b', 'armv8l']:
+            return 'arm64'
+        return 'amd64'
+
     def getKubernetesValues(self):
         """
         Returns a dict of Kubernetes component versions and paths for formatting into Kubernetes-related templates.
         """
         return dict(
+            ARCHITECTURE=self.getArchitecture(),
```

Comment:
This not always going to be the same result as `self._architecture` on the AWSProvisioner. So if you're on an and64 machine running `launch-cluster` to launch an ARM cluster, you'll get an ARM instance and an ARM AMI and then try to install amd64 Kubernetes on it.

Maybe `AWSProvisioner` should override `getArchitecture()` to return that? Or maybe we should make `AbstractProvisioner` have an `_architecture` and in the `__init__` set `self._architecture = self.getArchitecture()` as a default?